core/management/commands/load_copyright_map.py

```diff
@@ -8,15 +8,15 @@
 from core.load_copyright_map import loadCopyrightMap
 from core.management.commands import configure_logging
 
-configure_logging("load_copyright_map_logging.config", "load_coyright_map.log")
+configure_logging("load_copyright_map_logging.config", "load_copyright_map.log")
 
 LOGGER = logging.getLogger(__name__)
 
 
 class Command(BaseCommand):
-    help = "Load copyrights maps"
+    help = "Add records to lccn-date-copyright maps table from input file."
```

Comment:
Is there any way to make a customized "long" help message?  If so, could you go into the format of the files this accepts?

We need something on the wiki either way, but getting it into the command itself would be ideal.  I'd like to see something that just briefly explains the TSV file format.